test/dsp.jl

```diff
@@ -64,6 +65,30 @@ end
     @test conv(complex.(a, ones(4)), complex(b)) ≈ complex.([1., 4., 8., 10., 7., 6.], [1., 3., 6., 6., 5., 3.])
 end
 
+@testset "conv2" begin
+    
+    a =[1. 2. 1.;
+        2. 3. 1.;
+        1. 2. 1.]  
+
+    # [1, 0;
+    #  2, 3]
+    
+    b = [3. 2;
+         0. 1.]
+    
+    expectation = [3.0 8.0 7.0 2.0;
+                   6.0 14.0 11.0 3.0;
+                   3.0 10.0 10.0 3.0;
+                   0.0 1.0 2.0 1.0]
+    @test conv2(a, b) == expectation
+    @test conv2(complex.(a, ones(3,3)), complex.(b)) == complex.(expectation,
```

Comment:
You don't need the `ones`, since you are broadcasting. You could simply do `complex.(a, 1)`.